### DISCUSSION TITLE
Meeting agenda 20th July

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#22] [Meeting Agenda]
Our next meeting is scheduled for Wednesday, July 20th, 16:00 UTC.

I'd like to propose the following agenda:

- Further discussion around the contribution process, and the high-level principles see #19 
- Discuss the roadmap for the Regnosys contribution https://github.com/finos/community/issues/197

As discussed via email, I'm keen that we consider the Regnosys contribution and make our recommendations to the FINOS Team and Governing Board by mid-August. With this in mind, I'm proposing the following rough plan

 - next 2 weeks
   - discuss this plan
   - further discussion on high-level contribution principles
   - arrange a call between the TSC and Regnosys for Q&A
 - weeks 2-4
   - agree and formalise the high-level contribution principles
   - further asynchronous discussion around Regnosys 
   - 17th Aug TSC Meeting - agree on our formal recommendation

**Comment by jonfreedman** [comment]
I struggle to follow the Rosetta proposal, clearly https://github.com/finos/technical-steering-committee/discussions/21 is very relevant here. My understanding of the onion diagram on https://rosetta-technology.io/open-source is that with this proposal there's no UI, no execution engine but that Mophir could be the execution engine and Legend the UI. If that's the case then we're in the weird situation where the best commercial outcome for one of our projects is to compete with two of our others. Will be interesting to discuss.

**Reply by ColinEberhardt** [reply]
My understanding is that the Rosetta DSL allows modelling of entities and logic, see: https://github.com/REGnosys/rosetta-dsl#what-is-the-rosetta-dsl

> The Rosetta DSL addresses those shortcomings by enabling the representation of data and business logic

Open sourcing of the UI is on the roadmap as stated in the contribution issue https://github.com/finos/community/issues/197

> clearly https://github.com/finos/technical-steering-committee/discussions/21 is very relevant here

Agree, funnily enough I added that principle as a direct result of this proposed contribution!

**Comment by stephengoldbaum** [comment]
I second the concern about contributing to FINOS if getting value requires purchasing non-open-source components.  On the whole, that should be prohibited.  On the other hand, it sounds like that is alleviated by integrating with existing FINOS projects.  Does this contribution help with that effort? Does not contributing impede those integrations (I think it does due to legal, etc.)?  So it's possible the only way to get to fully open-source is to contribute to FINOS first.

The other point for me is principle #17 .  With Open Reg Tech SIG or other similar efforts. it's possible that we have regulations implemented across different projects.  Is that an issue?  Do we wait to deal with it then?
